birdhouse/scripts/check-instance-ready

```diff
@@ -13,29 +13,31 @@ COMPOSE_DIR="`dirname "$THIS_DIR"`"
 
 if [ -f "$COMPOSE_DIR/env.local" ]; then
     # Get PAVICS_FQDN
-    . $COMPOSE_DIR/env.local
+    . "${COMPOSE_DIR}/env.local"
 fi
 
+MONITOR_URL="https://${PAVICS_FQDN}/canarie/node/service/status"
+
```

Comment:
One thing we could do is use `/status` with the `Accept: application/json` to better validate the contents.
I think the 200 is returned only for the HTML representation (though the correct code could be returned in that case as well...) 

Technically, `/stats` could be completely empty if it never (yet) ran the log parsing cron job.
I've encountered this issue recently in the PR pipeline where tests were started too early because `/stats` looked OK as everything was empty.